graylog2-server/src/main/java/org/graylog2/inputs/codecs/GelfCodec.java

```diff
@@ -53,6 +53,7 @@
 public class GelfCodec extends AbstractCodec {
     private static final Logger log = LoggerFactory.getLogger(GelfCodec.class);
     private static final String CK_DECOMPRESS_SIZE_LIMIT = "decompress_size_limit";
+    public static final String NAME = "gelf";
```

Comment:
Is this needed for anything?

Reply:
the signature for adding a codec into a plugin module is `protected void addCodec(String name, Class<? extends Codec> codecClass)` . I `NAME` attribute was missing in the GelfCodec and hence I added it, part of this PR. 
After speaking with @bernd , I think we will need a new codec called `OktaCodec` for the OktaInput we are developing. Not sure, if we can add any Codec into a plugin module, without a `name` attribute . Let me know what you think ?

Reply:
You are correct about the `NAME` constant for plugins. Thanks for clarifying :) It will be needed. I registration in the plugin module will also be needed like this:

https://github.com/Graylog2/graylog-plugin-integrations/blob/6c53151852e0332026d2075d9591fef52f18fccf/src/main/java/org/graylog/integrations/IntegrationsModule.java#L99